test/functional/feature_fee_estimation.py

```diff
@@ -158,8 +140,10 @@ def send_tx(node, utxo, feerate):
     fee = tx_size * feerate
 
     tx = CTransaction()
-    tx.vin = [CTxIn(COutPoint(int(utxo["txid"], 16), utxo["vout"]), SCRIPT_SIG[utxo["vout"]])]
-    tx.vout = [CTxOut(int(utxo["amount"] * COIN) - fee, P2SH_1)]
+    tx.vin = [
+        CTxIn(COutPoint(int(utxo["txid"], 16), utxo["vout"]), REDEEM_SCRIPT)
+    ]
+    tx.vout = [CTxOut(int(utxo["amount"] * COIN) - fee, P2SH)]
```

Comment:
This change is in "split coins in a single tx in fee estimation test" but should be in "use a single p2sh script in fee estimation test" instead, yes?